mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -98,10 +98,10 @@
   "An ACL for managing access to ingest management functions."
   "INGEST_MANAGEMENT_ACL")
 
-;; Email subscription management acl
-(def email-subscription-management
+;; subscription management acl
+(def subscription-management
   "An ACL for managing access to email subscription functions."
```

Comment:
I think email should be removed in this description as well

Reply:
I think it doesn't hurt to leave it there because it is for email subscription. It's just that in ACL name, for some reason, they don't like EMAIL to be there.

Reply:
Dana's comment in the first demo was actually that we should remove all reference to 'email' for subscriptions because there can be other types of notifications that a user will receive and we don't want to be boxed into this one notification type. I forget some of the other ideas, but I think RSS was one. It's not a huge deal, but might be easier to remove 'email' here now, instead of having to remember to remove it in the future when we add more notification types.

Reply:
Okay. Removed.